python/mxnet/test_utils.py

```diff
@@ -134,6 +134,29 @@ def assert_almost_equal(a, b, threshold=None):
         raise Exception(msg)
     return rel
 
+def almost_equal_combo(a_src, b_src, threshold=None):
```

Comment:
consider follow numpy interface and use atol & rtol
https://docs.scipy.org/doc/numpy-dev/reference/generated/numpy.allclose.html#numpy.allclose